.gitignore

```diff
@@ -9,4 +9,5 @@
 *.lk
 *.int
 *.backup
-*.int
\ No newline at end of file
+*.int
+/*.dme
```

Comment:
I fudged up and need to remove this gonna work out what I did and undo it then sort that